build.gradle

```diff
@@ -233,9 +234,10 @@ distributions {
 
 docker {
   javaApplication {
-    baseImage = 'openjdk:8-jre-slim'
+    // https://bmuschko.github.io/gradle-docker-plugin/current/user-guide/#extension_2
+    baseImage = 'openjdk:11.0.15-jre-slim'
```

Comment:
Why use a Java 11 base image if classes are compiled for Java 17?

Reply:
Earlier I tried to run docker plugin with eclipse-temurin 17 image that had a lot of problems. So as per the documentation I used java 11 jre-slim (thinking that docker plugin only supports up to java 11) and it worked. 

Now that you pointed out so I searched for jre-slim 17 and the tasks builds with it too.